backend/routes/PostReaction/AttendeeHelperFunction.py

```
@@ -0,0 +1,59 @@
+def update_or_create_rsvp(db: Session, event_id: int, user_id: int, status: str) -> EventAttendee:
+    """Update an existing RSVP or create a new one."""
+    rsvp = get_user_rsvp(db, event_id, user_id)
+    if rsvp:
+        rsvp.status = status
+    else:
+        rsvp = EventAttendee(event_id=event_id, user_id=user_id, status=status)
+        db.add(rsvp)
+    db.commit()
+    return rsvp
+
+def delete_rsvp(db: Session, event_id: int, user_id: int):
```

Ruff (0.8.2)

Missing return type annotation for public function `delete_rsvp`

Add return type annotation: `None`

(ANN201)